git_machete/cmd.py

```diff
@@ -32,17 +32,31 @@ def __str__(self):
 RED = '\033[91m'
 
 
+ascii_only = False
+
+
 def bold(s):
-    return BOLD + s + ENDC
+    return BOLD + s + ENDC if not ascii_only else s
 
 
 def dim(s):
-    return DIM + s + ENDC
+    return DIM + s + ENDC if not ascii_only else s
 
 
 def underline(s):
-    return UNDERLINE + s + ENDC
+    return UNDERLINE + s + ENDC if not ascii_only else s + " *"
+
+
+def colored(s, color):
+    return color + s + ENDC if not ascii_only else s
+
 
+def vertical_bar():
+    return u"│" if not ascii_only else "|"
```

Comment:
Pls check if all this stuff (esp. wrt. unicode characters) works as expected under both Python 2.7 and 3.6